### PR TITLE
TEST

### PR DESCRIPTION
**Pull request by rlu39**

### CODE REVIEW

**Comment by ruiminglu** [comment]
why

**Comment by rlu39** [comment]
This is just a test

File: README [hunk 1]
```diff
@@ -1,2 +1,3 @@
 My Project
 On master/testing
+TEST PULL REQUEST
```
Comment (ruiminglu) [inline README]: what?

**Comment by ruiminglu** [comment]
haha

**Comment by ruiminglu** [comment]
Merged in

**Comment by ruiminglu** [comment]
Test notification

**Comment by rlu39** [comment]
OK

**Pull request merged**